dvc/remote/base.py

```diff
@@ -252,7 +252,9 @@ def load_dir_cache(self, checksum):
         try:
             with self.cache.open(path_info, "r") as fobj:
                 d = json.load(fobj)
-        except (ValueError, FileNotFoundError):
+        except FileNotFoundError:
+            raise
+        except ValueError:
             logger.exception("Failed to load dir cache '{}'".format(path_info))
```

Comment:
But our intention was to remove this logger.exception and raise something like BrokenDirCacheError here. Or just
```
...
except (ValueError, FileNotFoundError) as exc:
    raise DirCacheError(cause=exc)
```
here and then catch DirCacheError where we care for it and process it accordingly.